frontend/test/services/GridServiceTest.scala

```diff
@@ -4,6 +4,7 @@ import model.Grid.GridResult
 import org.specs2.mutable.Specification
 import utils.Resource
 import model.GridDeserializer._
+import com.netaporter.uri.dsl._
```

Comment:
Yeah, surprisingly, it _is_ required, the type of `mainImageUrl` has changed to `Option[Uri]` from `Option[String]`, and the implicit conversion is required for things to compile